cpp/ql/test/library-tests/members/this/this.ql

```diff
@@ -0,0 +1,6 @@
+import cpp
+
+query predicate thisExprType(ThisExpr e, Type t) { t = e.getType() }
```

Comment:
Is `thisExprType` needed?

Reply:
There didn't seem to be any existing tests for `ThisExpr`, so I thought it prudent to add it as well.